docs/1.0/critical/index.md

```diff
@@ -1211,8 +1211,8 @@ The tradition is unambiguous, but the editor deems it useless and prints an emen
 ``` xml
 Utrum fides
 <app>
-  <lem type="conjecture-corrected">sit</lem>
-  <rdg wit="#P #V #L"><corr>servus</corr></rdg>
```

Comment:
Right! 

However, at the DLL meeting I was thinking about this a little bit more. When you originally wrote this section, you did not add the `<corr>` tags. 

I then went back and added the `<corr>` to make this encoding parallel pattern to the use of `<supplied>` and `<surplus>`.

While I like this parallel encoding, isn't it also true that a "conjecture-corrected" type is identical to a "variation-substance" with the exception that instead of a `@wit` attribute, it should take a `@source` attribute (or if the current editor is the `@source` then it should not have an `@wit` or an `@source` at all)?

If we prefer to see this pattern as closely parallel to the "variation-substance" type, then maybe I was wrong to add in the `<corr>` tag. After all, it is obvious that this is an editor's correction when there is no `@wit` attached. 

I also think the DLL editors found it a bit annoying to have to add in the `<corr>` tag every time an editor or `@source` offered a reading not found in any of the surviving manuscript witnesses.

Let me know what you think.